src/darwin/Framework/CHIP/CHIPCluster.mm

```diff
@@ -17,6 +17,7 @@
 
 #import "CHIPCluster_internal.h"
 #import "CHIPDevice.h"
+#import "SpanUtils.h"
```

Comment:
Any chance to avoid using utils/helpers/support in naming? It makes reading and when to include non-obvious and they become a catch-all of mixed code and purpose.

In this case we convert things only, so maybe `SpanConversion.h` or similar?

Reply:
Or `NSDataSpanInterop.h` or anything that makes its purpose clearer.

Reply:
Per discussion with some people, will do `NSData+Span.h` and a class extension for `NSData` so the APIs will be:
```
[myData asByteSpan]
[NSData fromByteSpan:span]
```

Reply:
Doing that causes linker issues per https://developer.apple.com/library/archive/qa/qa1490/_index.html so looking into options.

Reply:
Switched to `NSDataSpanConversion.h`.